src/config/locales/en/error.js

```diff
@@ -79,6 +79,21 @@ export const error = {
         'Please confirm the date is correct by checking the box below.'
       ],
       note: '',
+      min: {
+        title: 'The applicant age is not approved',
+        message: [
+          'Your date of birth indicates you are over the age of 100.',
```

Comment:
Let's try adding this line.
```
Your date of birth indicates you are over the age of 100. Please contact your security office representative or the individual who initiated your investigation regarding further instruction.
```

See [this](https://gsa-tts.slack.com/archives/C9VSX3JNM/p1568658106033800?thread_ts=1568657409.032200&cid=C9VSX3JNM)